develop/devguide/GQI Extensions/API_Reference/IGQIUpdater.md

```diff
@@ -19,7 +19,7 @@ It allows adding, removing, and updating rows, or updating individual cells.
 
 ### void AddRow(GQIRow)
 
-Adds a new row to the query result.
+Adds a new row to the query result. If a row with the same row key already exists, all cells will be updated instead. This method effectively acts as an *add or update* mechanism.
```

Comment:
Note that it not only updates cells but also metadata, and possibly other row attributes in the future.

```suggestion
Adds a new row to the query result. If a row with the same row key already exists, that row will be updated instead. This method effectively acts as an *add-or-update* operation.
```